sa/_db/migrations/20150904121736_CertificateStatusNotNullManyFields.sql

```diff
@@ -0,0 +1,23 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+ALTER TABLE `certificateStatus` MODIFY `subscriberApproved` tinyint(1) NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `status` varchar(255) NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `ocspLastUpdated` datetime NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `revokedDate` datetime NOT NULL;
```

Comment:
Just to be super precise, the code talks about the zero value of time.Time (Go concept) rather than a NULL value (SQL concept):

> // revokedDate: If status is 'revoked', this is the date and time it was
> //   revoked. Otherwise it has the zero value of time.Time, i.e. Jan 1 1970.
> RevokedDate time.Time `db:"revokedDate"`

I think I wrote that code, and I don't think I had a specific reason for avoiding a pointer. I think my lingering C-oriented hindbrain still thinks of pointers as something you have to allocate and deallocate manually and therefore a big pain. I'd be okay with changing the representation, but keep in mind that `RevokedDate` is basically undefined unless `Status` is set to `"revoked"`, in which case `RevokedDate` will be a non-zero value.